07-routing-gossip.md

```diff
@@ -279,7 +279,7 @@ The following `address descriptor` types are defined:
    * `3`: Tor v2 onion service; data = `[10:onion_addr][2:port]` (length 12)
```

Comment:
Should we just replace this with `Deprecated (length 12). Used to contain Tor v2 onion services`? No need to have a bunch of docs about it.